core/templates/dev/head/components/social_buttons/social_buttons_directive.html

```diff
@@ -38,40 +38,40 @@
 
 <style>
   .oppia-footer-social .oppia-footer-icons {
-        display: -webkit-inline-box;
-        font-size: 24px;
-        position: relative;
-        top: -10px;
-    }
+    display: -webkit-inline-box;
+    font-size: 24px;
+    position: relative;
+    top: -10px;
+  }
 
-    .oppia-footer-icons .google-plus-color {
-        background: #dd4b39;
-        margin-right: 6px;
-        padding: 5px 7px;
-    }
-    .oppia-footer-icons .youtube-color {
-        background: #cd201f;
-        margin-right: 6px;
-        padding: 5px 9px;
-    }
-    .oppia-footer-icons .fb-color {
-        background: #3b5998;
-        margin-right: 6px;
-        padding: 5px 10px;
-    }
-    .oppia-footer-icons .twitter-color {
-        background: #1da1f2;
-        margin-right: 6px;
-        padding: 5px 6px;
-    }
-    .fb-color i.fa.fa-facebook {
-    bottom: -6px;
+  .oppia-footer-icons .Google.Plus {
+    background: #dd4b39;
+    margin-right: 6px;
+    padding: 5px 7px;
+  }
+  .oppia-footer-icons .Youtube {
+    background: #cd201f;
+    margin-right: 6px;
+    padding: 5px 9px;
+  }
+  .oppia-footer-icons .Facebook {
+    background: #3b5998;
+    margin-right: 6px;
+    padding: 5px 10px;
+  }
+  .oppia-footer-icons .Twitter {
+    background: #1da1f2;
+    margin-right: 6px;
+    padding: 5px 6px;
+  }
+  .Facebook i.fa.fa-facebook {
```

Comment:
@seanlip Sir, 
Found the [error](https://travis-ci.org/oppia/oppia/jobs/212068496#L6608) at line number 6608 in the logs. It says "ERROR:  test_signup_page_does_not_have_top_right_menu (core.controllers.profile_test.SignupTest)" as well as IndexError: Body contains bad string '>Sign in'

In adopting font awesome we removed **Sign In with Google Image** and replaced with this PRs' code causing this error.
What might be the process now?